docs/src/man/manipulation_functions.md

```diff
@@ -0,0 +1,1345 @@
+# Data Frame Manipulation Functions
+
+The seven functions below can be used to manipulate data frames
```

Comment:
Also here, as I have commented - maybe add a sentence at the top explaining the teaching approach of this material. Thank you!